### PR TITLE
Task/sdk sample app and documentation

### PR DESCRIPTION
**Pull request by Blu3m00n11**
### What Does This PR Do?
This PR is for the Sample app and Documentation for the SDK unity package. It is documentation that explains what the sample app is and what it is used for.

### Where Should The Reviewer Start?
From the top to the bottom of the page.

### How Should This Be Manually Tested?
Reading threw the documentation to make sure it is covering all of the needed information.

### CODE REVIEW

**Review by Debadri-Redapple — APPROVED** [review]

**Review by drxeno02 — APPROVED** [review]
@Blu3m00n11 This is a great start! 💯 

I will expand it and fill in some missing parts later. Thank you for making the documentation more clear.

**Review by noah1013 — COMMENTED** [review]
Maybe there's a reason for it or maybe you are going to remove one of them, but the description that you guys made in the "What is Mikros?" section is extremely similar to the description for Mikros at the bottom. If you are removing one of them forget this comment.

For the FAQ, hopefully, you have a list or some sort of idea of what questions are going to be put in the section. Otherwise, if you have any questions about the SDK, whether about integration, how data markers work, or more, they should be put there.

Otherwise great work so far 👍. Hope to see the progress you make in the future!

**Review by noah1013 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * [How to Join MIKROS Community Slack?](*how-to-join-mikros-community-slack)
 * [What Are the Rules For The MIKROS Community Slack?](*what-are-the-rules-for-the-mikros-community-slack)
 * [FAQ](*faq)
+* [SAMPLE APP & DOCUMENTATION](#sample-app-&-documentation)
```
[the remark this answers is not in the document]
Comment (Blu3m00n11) [reply README.md:10]: Thank you for the feed back I  have made these changes.

File: README.md [hunk 2]
```diff
@@ -100,3 +101,24 @@ Supply the following URL:
 ```
 https://github.com/TATUMGAMES/TG-MIKROS-SDK-unity-package.git
 ```
+
+<a name="sample-app-&-documentation"></a>
+## Sample App & Dcumentation
+
+* Authentification
+
```
[the remark this answers is not in the document]
Comment (Blu3m00n11) [reply README.md]: Thank your for the feedback, it is really helpful. I was just laying everything out. I will get the format corrected and my screenshots resubmitted the correct way. Thank you so much

[the remark this answers is not in the document]
Comment (drxeno02) [reply README.md]: @Blu3m00n11 Take some time and master markdown language. Ref- https://www.markdownguide.org/basic-syntax/#images-1

**Comment by Debadri-Redapple** [comment]
@Blu3m00n11 @noah1013 @drxeno02 The screenshots for our sample app should not be present in this repo. This whole git repo is being distributed to users as a package and we can't refer to our sample app here. Also any mention of the sample app in the Readme is not required.
However other changes in this Readme are great.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,106 @@
...
+5. *(Optional)* Mikros collects and sends session data for better user insights and is also crucial for Analytics purpose. This can be enabled or disabled by the ***Auto Track User Session*** option from the Mikros Settings. By default, it is kept enabled and is recommended to keep it that way.
+6. *(Optional)* Mikros collects and sends metadata for better user insights and is also crucial for Analytics purpose. This can be enabled or disabled by the ***Auto Track User Metadata*** option from the Mikros Settings. By default, it is kept enabled and is recommended to keep it that way.
+
+<a name="sample-app-&-documentation"></a>
+## Sample App & Dcumentation
+
+* Authentification
+    
+    
+ This is the Mikros Sample App main screen. The Authentication button will allow you to be authenticated and allow you to use the Mikros Sample App.
```
Comment (Debadri-Redapple) [inline README.md:49]: @noah1013 @Blu3m00n11 @drxeno02 The integrators won't be provided with our sample app. So anything related to it should be removed.

**Review by Debadri-Redapple — CHANGES_REQUESTED** [review]
@Blu3m00n11 @noah1013 @drxeno02 The screenshots for our sample app should not be present in this repo. This whole git repo is being distributed to users as a package and we can't refer to our sample app here. Also any mention of the sample app in the Readme is not required.
However other changes in this Readme are great.

File: README.md (hunk 3 above)
Comment (drxeno02) [reply README.md:49]: @Debadri-Redapple Yes, exactly. I was going to redo this README after they submit. This should be a guide on how to use MIKROS. I think I will have to be the one to finalize everything. For now, the work here is about 90% complete. I will finish everything else off.

We'll let this PR sit until I can get to it.

**Review by drxeno02 — CHANGES_REQUESTED** [review]
I will be re-writing this README. Let this PR sit for a bit. Everyone can move on for now.

**Review by drxeno02 — APPROVED** [review]

**Review by Debadri-Redapple — CHANGES_REQUESTED** [review]
@drxeno02 @Blu3m00n11 @noah1013 @yongyuthbianca  The pictures related to Sample app inside the Documentation folder should be removed.

File: README.md (hunk 3 above)
Comment (Debadri-Redapple) [reply README.md:49]: @drxeno02 This Sample App & Documentation section needs to be removed from Readme, before we can merge this PR.

**Review by Debadri-Redapple — CHANGES_REQUESTED** [review]

**Comment by drxeno02** [comment]
@Debadri-Redapple Regarding your comments here: https://github.com/TATUMGAMES/TG-MIKROS-SDK-unity-package/pull/9#discussion_r820385141

Yes, you are correct. And you can remove it yourself. There is no concept of "your code, my code". If you see something wrong, jump on that branch and fix it, then commit/push. Make the necessary updates. Remember, these are Interns who are still learning. Help them out. Thanks!

**Comment by Debadri-Redapple** [comment]
@drxeno02 Edits made in this PR is not required for the package. Hence closing this.

**Pull request closed without merging**